test/e2e-go/cli/goal/expect/goalNodeStatusTest.exp

```diff
@@ -18,6 +18,9 @@ if { [catch {
     set NETWORK_TEMPLATE "$TEST_DATA_DIR/nettemplates/TwoNodes50Each.json"
 
     # Create network
+    ::AlgorandGoal::CreateNetwork $NETWORK_NAME $NETWORK_TEMPLATE $TEST_ALGO_DIR $TEST_ROOT_DIR
```

Comment:
Consider adding CreateAndStartNetwork proc to combine these steps.

Reply:
I'm not opposing to having a utility function that would combine the two, but I don't feel that the "value" is there.
i.e. given that this is a goal-oriented test, we need to use the tests to invoke goal commands.. if the command structure is too complicated, it implies that our goal API isn't design well.